sumologic/provider.go

```diff
@@ -14,6 +14,7 @@ import (
 )
 
 func Provider() terraform.ResourceProvider {
+	log.Printf("Sumo Logic Terraform Provider Version=%s\n", ProviderVersion)
```

Comment:
do we still want the log line?

Reply:
Yeah, I want that. This would help us identify the provider version when we ask customers for logs.